automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/previewmodal/PreviewSpec.scala

```diff
@@ -124,7 +124,7 @@ class PreviewSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpe
     }
   }
 
-  "Preview Modal should display correct message and preview when file is accessible for dos:// link" ignore {
+  "Preview Modal should hide preview even when dos:// link resolves to an otherwise previewable file" in {
```

Comment:
Does this read clearer:  "Preview Modal should hide preview for all dos:// links"